include/filters/median.hpp

```diff
@@ -46,10 +43,11 @@
  * Algorithm from N. Wirth's book, implementation by N. Devillard.
  * This code in public domain.
  */
-#define ELEM_SWAP(a,b) { register elem_type t=(a);(a)=(b);(b)=t; }
+#define ELEM_SWAP(a, b) {register elem_type t = (a);(a) = (b);(b) = t;}
```

Comment:
Nit: instead of this, can we use [std::swap](https://en.cppreference.com/w/cpp/algorithm/swap)?  Sorry I missed it in my last review.

Reply:
Done in https://github.com/sloretz/filters/pull/1/commits/143d7f1e60b4fed049485b8ab6c86eb7928c25c5.